cmd/token_create.go

```diff
@@ -0,0 +1,85 @@
+package cmd
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/MakeNowJust/heredoc/v2"
+	"github.com/spf13/cobra"
+	lab "github.com/zaquestion/lab/internal/gitlab"
+)
+
+var (
+	expiresAt time.Time
+)
+
+var tokenCreateCmd = &cobra.Command{
+	Use:   "create",
+	Short: "create a new token",
```

Comment:
```suggestion
	Short: "create a new personal access token",
```
I don't want people to be confused with the project/groups.